addons/account_peppol/views/res_config_settings_views.xml

```diff
@@ -134,13 +134,7 @@
                                            nolabel="1"/>
                                 </div>
                                 <div class="mt-4" invisible="account_peppol_proxy_state != 'not_registered'">
-                                <field name="account_peppol_mode_constraint" invisible="1"/>
```

Comment:
Keep it. You must never remove a field from a view because someone could have build an extension of the view and use the field in a xpath.

Reply:
Ok thank you